src/shared-components/SidebarBase/SidebarBase.module.css

```
@@ -366,13 +364,13 @@
   }
 
   100% {
-    left: var(--space-0);
+    left: 0;
   }
 }
 
 @keyframes goToLeftBigScreen {
```

Stylelint (17.3.0)

[error] Expected keyframe name "goToLeftBigScreen" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)